bciers/apps/reporting/src/app/components/products/ProductionDataForm.tsx

```diff
@@ -97,6 +97,8 @@ const ProductionDataForm: React.FC<Props> = ({
       onChange={(data) => onChange(data.formData)}
       continueUrl={navigationInformation.continueUrl}
       errors={errors}
+      saveButtonDisabled={errors && errors.length > 0}
+      submitButtonDisabled={errors && errors.length > 0}
```

Comment:
`onChange` doesn't update the errors, so this means that if the server returns errors once, we won't be able to submit anymore. I don't think we need those 2 lines, they weren't part of the AC for your card anyways